src/app/board/card.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { FaGithub, FaLinkedin } from 'react-icons/fa'; 
+const Card = ({ title, description, github, linkedin }) => {
+  return (
+    <div className="card">
```

Comment:
these classnames aren't defined, you have to use tailwind to get the layout right, but the html looks good so its most likely just a matter of styling